py/tests/core/test_config_schema.py

```diff
@@ -0,0 +1,77 @@
+from configu import ConfigSchema
+from configu.core.config_schema import ConfigSchemaDefinition
+import pytest
+
+@pytest.mark.parametrize("path", (
+  "tests/staticfiles/config_schemas/valid/empty.cfgu.json",
+  "tests/staticfiles/config_schemas/valid/basic.cfgu.json",
+  "tests/staticfiles/config_schemas/valid/complex.cfgu.json",
+))
+def test_valid_config_schemas(path):
+    ConfigSchema.parse(ConfigSchema(path))
+
+@pytest.mark.parametrize("path", (
+    "tests/staticfiles/config_schemas/invalid/regex_missing_pattern.cfgu.json",
+    "tests/staticfiles/config_schemas/invalid/reserved_names.cfgu.json",
+    "tests/staticfiles/config_schemas/invalid/default_and_required.cfgu.json",
+    "tests/staticfiles/config_schemas/invalid/default_and_template.cfgu.json",
+    "tests/staticfiles/config_schemas/invalid/nonexistent_type.cfgu.json",
+    "tests/staticfiles/config_schemas/invalid/invalid_default.cfgu.json",
+))
+def test_invalid_config_schemas(path):
+    with pytest.raises(ValueError):
+        ConfigSchema.parse(ConfigSchema(path))
+
+@pytest.mark.parametrize("path", (
+    # "tests/staticfiles/config_schemas/invalid/bad_extension.json", // TODO: This currrent fails. Is this a bug?
```

Comment:
@RonConfigu @rannn505 Wasn't sure if this was a bug or not. As far as I was aware we should be throwing a `ValueError` if trying to use a config schema without the extension `.cfgu.<<ext>>`.
If so we should probably raise that as a bug, and I'll keep this test commented out for now.
If not, I'll remove this comment :+1:

Reply:
Correct 💯
Unit tests justifying their purpose, once again.
This will fix it:
config_schema.ph - line 69
```python
EXT = " | ".join(["".join(ext) for ext in zip([CFGU.EXT], TYPES.keys())])
```
removed `cycle` and added `[]`to `CFGU.EXT`. There was a time when I though CFGU.EXT will be a list and this is a leftover. Thanks for catching 🥇 

You could solve this in a different way if you choose.